_includes/head.html

```diff
@@ -19,4 +19,5 @@
 	link.type = 'text/css';
 	link.href = window.location.search.match( /print-pdf/gi ) ? 'css/print/pdf.css' : 'node_modules/reveal.js/css/print/paper.css';
 	document.getElementsByTagName( 'head' )[0].appendChild( link );
-</script>
\ No newline at end of file
+  console.log("Test");
```

Comment:
delete this

Reply:
sure I will delete. thanks for finding bug